articles/quickstart/spa/vuejs-beta/interactive.md

```diff
@@ -0,0 +1,179 @@
+
+```bash
+cd <your-project-directory>
+npm install @auth0/auth0-vue@beta
+```
+
+### Register the plugin
+
+For the SDK to function, you must register the plugin with your Vue application using the following properties:
+
+- `domain`: The domain of your Auth0 tenant. This value is in the Auth0 Dashboard under your Application's Settings in the Domain field. If you are using a [custom domain](https://auth0.com/docs/custom-domains), set this to the value of your custom domain instead.
+- `clientId`: The ID of the Auth0 Application you set up earlier in this quickstart. Find this in the Auth0 Dashboard under your Application's Settings in the Client ID field.
+- `authorizationParams.redirect_uri`: The URL in your application that you would like Auth0 to redirect users to after they have authenticated. This corresponds to the callback URL you set up earlier in this quickstart. This value is in the Auth0 Dashboard under your Application's Settings in the Callback URLs field. Make sure what you enter in your code matches what you set up earlier or your users will see an error.
+
+The plugin will register the SDK using both `provide` and `app.config.globalProperties`. This enables both the [Composition API](https://vuejs.org/guide/introduction.html#composition-api) and [Options API](https://vuejs.org/guide/introduction.html#options-api).
+
+::::checkpoint
+
+:::checkpoint-default
+
+The plugin is now configured. Run your application to verify that:
+- the SDK is initializing correctly
+- your application is not throwing any errors related to Auth0
+
+:::
+
+:::checkpoint-failure
+If your application did not start successfully:
+* Verify you selected the correct application
+* Save your changes after entering your URLs
+* Verify the domain and Client ID imported correctly
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+
+:::
+::::
+
+## Add login to your application {{{ data-action=code data-code="login.js" }}}
+
+Next, you will set up login for your project. You will use the SDK’s `loginWithRedirect` function exposed on the return value of `useAuth0`, which you can access in your component's setup function. It will redirect users to the Auth0 Universal Login page. and, after a user authenticates, redirect then back to the callback URL you set up earlier in this quickstart.
+
+### Using the Options API
+If you are using the Options API, you can use the same `loginWithRedirect` method from the global `$auth0` property through the `this` accessor.
+
+
+::::checkpoint
+
+:::checkpoint-default
+
+You should now be able to log in using Auth0 Universal Login.
+
+Click the login button and verify that:
+* your Vue application redirects you to the Auth0 Universal Login page
+* you can log in or sign up
+* Auth0 redirects you to your application using the value of the `redirect_uri` you used to configure the plugin.
```

Comment:
Do you think we should consistently refer to `authorizationParams.redirect_uri` throughout the QS?

Reply:
Ye, I would